src/backend/distributed/commands/table.c

```diff
@@ -1060,11 +1060,13 @@ ErrorIfUnsupportedAlterTableStmt(AlterTableStmt *alterTableStatement)
 			case AT_EnableTrigAll:
 			case AT_DisableTrigAll:
 			case AT_ReplicaIdentity:
+			case AT_ValidateConstraint:
 			{
 				/*
 				 * We will not perform any special check for ALTER TABLE DROP CONSTRAINT
 				 * , ALTER TABLE .. ALTER COLUMN .. SET NOT NULL and ALTER TABLE ENABLE/
-				 * DISABLE TRIGGER ALL, ALTER TABLE .. REPLICA IDENTITY ..
+				 * DISABLE TRIGGER ALL, ALTER TABLE .. REPLICA IDENTITY .., ALTER TABLE
```

Comment:
Should update the `errdetail ` below to add `VALIDATE `?

Reply:
I updated the `errdetail` to include `VALIDATE CONSTRAINT` support

Reply:
changing the error messages changed the outputs of other tests. I need to make changes to several other test outputs to sort this out.